tex/pgfcircbipoles.tex

```diff
@@ -1718,14 +1718,21 @@
 
 %% Easily usable styles
 
+\ctikzset{o-o/.style =  {\circuitikzbasekey/bipole/nodes/left=empty, \circuitikzbasekey/bipole/nodes/right=empty}}
```

Comment:
Oh, one space too much after equal sign. I;m sorry about that, I'll fix that in next patch.